ssr/progress-bar.js

```diff
@@ -22,10 +30,12 @@ export default class ProgressBar {
 
   update(current) {
     this.current = current;
-    this.draw(this.current / this.total);
+    this.draw();
   }
 
-  draw(currentProgress) {
+  getRow() {
```

Comment:
More exactly known as `getOut` but that doesn't sound nice. Name suggestion is greatly appreciated 😁